app/services/AgentRelationshipService.scala

```diff
@@ -52,22 +52,37 @@ class AgentRelationshipService @Inject()(
         agentOrgId: Long,
         organisationId: Long,
         individualId: Long,
-        isAgent: Boolean)(implicit hc: HeaderCarrier): Future[Unit] =
-    Future
-      .traverse(pLinkIds)(pLink => appointAgent(pLink, agentOrgId, organisationId, individualId, isAgent))
-      .map { x =>
-        // Introduce a delay here to give modernised a chance to process its queue
-        // This is a temporary measure to avoid duplicating agent requests when appointing multiple agents
-        // TODO https://jira.tools.tax.service.gov.uk/browse/VTCCA-2972
-        Thread.sleep(config.agentAppointDelay * 1000)
+        isAgent: Boolean,
+        agentCode: Long)(implicit hc: HeaderCarrier): Future[Unit] = config.newAppointAgentJourneyEnabled match {
+    case true =>
+      assignAgentToSomeProperties(AgentAppointBulkAction(agentCode = agentCode, propertyLinkIds = pLinkIds)).map(_ =>
```

Comment:
appoint/assign

Reply:
AgentAppointBulkAction is the old model. I'd prefer not to touch the old stuff. Im just using it as it